modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -167,4 +167,63 @@ public static int hash(long key) {
 
         return hash(val);
     }
+
+    /**
+     * Constructs {@code long} from byte array.
+     *
+     * @param bytes Array of bytes.
+     * @param off Offset in {@code bytes} array.
+     * @return Long value.
+     */
+    public static long bytesToLong(byte[] bytes, int off) {
```

Comment:
I think we should move all method like this to class with more meaningful name. E.g. ByteUtils, Bytes, etc.
Otherwise, `IgniteUtils` will turn into a garbage.